### ISSUE TITLE
This isn't actually an issue :P

### ISSUE THREAD

**Issue by cj81499** [TimesNewBroman/Theming-Stuff#1]
Hey TimesNewBroman. If you want to turn this into a repo, let me know and I'd be glad to help. If you don't, but you want these on a repo, let me know and I'll put them on https://cj81499.github.io

**Comment by lushm0nstera** [comment]
I'd actually like to know more.  I'm not sure if I'll actually do it, but for the sake of simplicity, making it a true repo would be useful.  Thanks for contacting me.

**Comment by cj81499** [comment]
Sure! It's a pretty easy process, if you want to check out my repo for use as a template, head here: 

The general process for getting something on the repo is to make the deb file, add information into the packages file and then compress it as a .gz file.

It's technically optional, but it's nice to have a depiction too, which is just basic HTML and CSS that  can be modified to match whatever package you want.

Making the .deb is the hardest part in my opinion. There are tutorials online, but they're hard to follow imo. The General idea is that you put your files into a folder as well as a control file and then run a command in the terminal to convert that into a .Deb.  I'm probably going to make a video tutorial of how to do this sometime.
